tests/jballerina-unit-test/src/test/resources/test-src/bala/test_bala/object/test_objects.bal

```diff
@@ -466,6 +466,14 @@ function testSubTypingForDifferentOrgNameAndVersionWithAllPublicFields() {
     assertTrue((<any> ob3) is foo2:ObjectWithPublicFieldsAndMethods);
 }
 
+readonly service class FooClass {
```

Comment:
Can we actually test something here? By creating a `FooClass` value and calling the method. May have to change service to client or have a normal method.